addons/payment/models/payment_acquirer.py

```diff
@@ -277,14 +277,34 @@ def _create_missing_journal_for_acquirers(self, company=None):
             journals += acquirer.journal_id
         return journals
 
+    @api.model
+    def _update_journal(self, vals):
+        '''
+        Update the payment method of the journal depending on the field 'payment_flow' and 'save_token'.
+        If these fields signify that the payment is electronic, add the electronic payment
+        method to the journal. Otherwise, remove the electronic payment method.
+        '''
+        method = self.env.ref('payment.account_payment_method_electronic_in')
+        if not method: return
+        for record in self:
+            if not record.journal_id: continue
+            if (vals.get('payment_flow', record.payment_flow) == 's2s' or
```

Comment:
I've no idea how to set those fields. How did you test?

Reply:
On the payment acquirer form, you can change the field "payment_flow" or "save_token" that will affect it's journal payment method when you save it. Should I write a unit test for this case? I haven't done it as the method is quite simple.